meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/edk2-firmware-tegra_%.bbappend

```diff
@@ -1,9 +1 @@
-
-FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
-
-SRC_URI += "file://0007-runtime-access-for-KernelCommandLine-efivar.patch;patchdir=.."
```

Comment:
Sorry for being late to the party. I hope, you don't mind me posting my question here. 

I figured that mender uses the KernelCommandLine variable for passing the machine ID to the "new" RootFS if the feature "mender-persist-systemd-machine-id" is active.
Is this feature supposed to work despite removing this patch?

Upstream, the variable is still not available to runtime:
https://github.com/NVIDIA/edk2-nvidia/blob/r36.4.0/Silicon/NVIDIA/Drivers/NvidiaConfigDxe/NvidiaConfigHii.vfr#L63

Without  runtime access to this variable, the script `efi_systemd_machine_id.sh` seems to fail persisting the machine ID with the message
```
efivar: Read-only file system
```

(I tested the script on a Jetpack 6.1 system, but I don't see a relevant difference to JP 5 code)